public/index.html

```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Developer Dao</title>
```

Comment:
```suggestion
    <title>Developer DAO</title>
```

Just for consistency across platforms. It's already named that way on Discord and here on GitHub.

Reply:
Done! Nice catch.